xml/gl.xml

```diff
@@ -24637,7 +24637,7 @@ typedef unsigned int GLhandleARB;
             <param group="SizedInternalFormat"><ptype>GLenum</ptype> <name>internalformat</name></param>
             <param><ptype>GLsizei</ptype> <name>width</name></param>
             <param><ptype>GLsizei</ptype> <name>height</name></param>
-            <param group="TexStorageAttribs">const int *<name>attrib_list</name></param>
+            <param group="TexStorageAttribs">const <ptype>GLint</ptype>* <name>attrib_list</name></param>
         </command>
         <command>
             <proto>void <name>TexStorageAttribs3DEXT</name></proto>
```

Comment:
I think the main concern in #498 was the function names.
```suggestion
            <proto>void <name>glTexStorageAttribs3DEXT</name></proto>
```
(and elsewhere)

Reply:
yikes. thanks.

Reply:
I also want the `int` to `GLint` change to happen as `GLint` is what is used everywhere. I didn't mention that in #498 but I did ask here about it https://github.com/KhronosGroup/OpenGL-Registry/pull/491#issuecomment-1047971998.